sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -13,6 +13,15 @@
 #include <CL/sycl/detail/common.hpp>
 #include <CL/sycl/detail/os_util.hpp>
 #include <CL/sycl/detail/pi.h>
+#include <cassert>
+
+// Function to load the shared library
+// Implementation is OS dependent. 
+void* loadOsLibrary(const char* library);
```

Comment:
this interface can take "const std::string &"

Reply:
done.